tests/ui/test_ui_tools.py

```diff
@@ -462,13 +462,23 @@ def test_keypress_GO_BACK(self, mocker, stream_view, key):
         stream_view.set_focus.assert_called_once_with("body")
         assert stream_view.log == self.streams_btn_list
 
-    @pytest.mark.parametrize('current_focus, stream', [
-        (0, 'FOO'),
-        (2, 'fan'),
-        (4, 'BOO'),
+    @pytest.mark.parametrize('current_focus, stream, \
+                             search_streams_key, go_back_key', [
+        (current_focus, stream, search_streams_key, go_back_key)
+        for current_focus, stream, search_streams_keys, go_back_keys in [
+            (0, 'FOO', keys_for_command('SEARCH_STREAMS'),
+                keys_for_command('GO_BACK')),
+            (2, 'fan', keys_for_command('SEARCH_STREAMS'),
+                keys_for_command('GO_BACK')),
+            (4, 'BOO', keys_for_command('SEARCH_STREAMS'),
+                keys_for_command('GO_BACK')),
```

Comment:
There's duplication here, as in the other commit, where I think we should just be able to loop over the same values each time.